gr-qtgui/python/qtgui/__init__.py

```diff
@@ -23,14 +23,22 @@
     __path__.append(os.path.join(dirname, "..", "..", "swig"))
     from .qtgui_swig import *
 
+try:
+	import matplotlib.pyplot as plt
+	from .distanceradar import DistanceRadar
+	from .azeplot import AzElPlot
+except ImportError:
+	from gnuradio import gr
+	gr.log.warn("Matplotlib is a required dependency to use DistanceRadar and AzElPlot."
+                "  Please install matplotlib to use these blocks (https://matplotlib.org/)")
+
+
```

Comment:
Not huge deal, but can you please remove extra line of whitespace

Reply:
Changes have been incorporated thank you :) @mormj for helping